File: coleccionesYDiccionarios/Ejercicio3.java
package coleccionesYDiccionarios;

import java.io.IOException;
import java.util.ArrayList;
import java.util.Collections;
import utiles.Teclado;

/**
 * Escribe un programa que ordene 10 números enteros introducidos por teclado y
 * almacenados en un objeto de la clase ArrayList.
 * 
 * @author Álvaro Leiva Toledano
 * @version 1.0
 */
public class Ejercicio3 {

	public static void main(String[] args) throws NumberFormatException, IOException {

		// Colección numeros.
		ArrayList<Integer> numeros = new ArrayList<Integer>();

		for (int i = 1; i <= 10; i++) {
			numeros.add(Teclado.leerEntero("Introduce el valor nº" + i + ":"));
		}

		System.out.println("ArrayList sin ordenar: " + numeros);
		
		Collections.sort(numeros);
		
		System.out.println("ArrayList ordenado: " + numeros);

	}

}

File: coleccionesYDiccionariospy/Ejercicio3.py
# -*- coding: utf-8 -*-
'''
Created on 3 abr. 2019
Escribe un programa que ordene 10 números enteros introducidos por teclado
y almacenados en un objeto de la clase ArrayList.

@author: Álvaro Leiva Toledano
@version: 1.0
'''

numeros = []
for i in range(10):
    numeros.append(int(input("Introduzca el número " + str(i+1) + " de 10: ")))

print("\nLista sin ordenar: " + str(numeros))

numeros.sort()

print("Lista ordenada: " + str(numeros))
